frontend/app/services/subscription-service.server.ts

```diff
@@ -111,25 +116,50 @@ function createSubscriptionService() {
 
     return userParsed;
   }
+
+  async function updateUser(userId: string, userInfo: UserUpdateInfo) {
+    const auditService = getAuditService();
+    auditService.audit('alert-subscription.get', { userId });
+
+    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}`);
+    const response = await instrumentedFetch(fetchFn, 'http.client.interop-api.users.patches', url, {
```

Comment:
```suggestion
    const response = await instrumentedFetch(fetchFn, 'http.client.cdcp-api.users.patches', url, {
```

Same comment can be applied throughout this file where `interop-api` is used in the metric name.

Reply:
fixed all instrumentedFetch messages